docs/index.rst

```diff
@@ -3,14 +3,24 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to open_plan's documentation!
-#####################################
+open_plan Documentation
+#######################
 
+.. image:: /logos/open_plan_logo.png
+   :scale: 10%
+   :align: right
 
 
+open_plan is a Python-based, open-source energy-system modeling tool targeted at simulating energy cells or mini-grids.
+
+It uses the framework `oemof <https://oemof.org/>`_ to describe and solve the energy system and will provide a user-friendly web-based interface to intereact with the python. Enabling users **without python or programming knowledge** to use the tool and simulate energy systems.
```

Comment:
```suggestion
It uses the `oemof <https://oemof.org/>`_ framework to describe and optimize the energy system and will provide a user-friendly web-based interface to intereact with python. This will enable users **without python or programming knowledge** to use the tool and simulate energy systems.
```